### PR TITLE
fix(stacktrace): in app v. system

### PR DESCRIPTION
**Pull request by doralchan**
**Before:**
Screenshot 2023-03-13 at 11 05 36 AM

**After:**
Screenshot 2023-03-13 at 11 05 44 AM

### CODE REVIEW

File: static/less/group-detail.less [hunk 1]
```diff
@@ -304,22 +304,14 @@ div.traceback > ul {
...
     }
 
     .title {
-      padding: 8px 20px;
+      padding: 6px 20px;
       font-size: 13px;
       margin: 0;
       line-height: 16px;
-      background: lighten(@blue-light, 35);
+      background: #faf9fb;
```
Comment (doralchan) [inline static/less/group-detail.less:315]: I'm only doing this because I don't want to add anything to the variables.less file

**Comment by malwilley** [comment]
@doralchan just wanted to check, were these height and text color changes for the collapsed "Called from" frame purposeful?

Before (shorter and lighter text):
CleanShot 2023-03-13 at 15 16 15

After (taller and dark text);
CleanShot 2023-03-13 at 15 16 23

**Comment by doralchan** [comment]
Thanks for the reminder @malwilley I went ahead and made it a little more consistent with what we have for mobile. I kept the gray text for the label but made the font size similar to the rest of the frames (removing custom 12px). Here are some quick screenshots: 

Screenshot 2023-03-13 at 3 39 41 PM

Screenshot 2023-03-13 at 3 29 56 PM

**Comment by malwilley** [comment]
Oh one more thing @doralchan - the blue frame background used to have a slight color change on hover, but it is gone with the changes here. Looks like the dark theme still works though.

**Review by malwilley — APPROVED** [review]

**Pull request merged**